### DISCUSSION TITLE
Learning pod 1

### DISCUSSION THREAD

**Discussion by EDCI338** [cmadland/edci338-202201#70] [General]
hey team! just starting this discussion to add your links

**Comment by miaharmitage** [comment]
https://miaharmitage.opened.ca
This is the link to my Wow and Wonder for week 1 :)

**Comment by tiannaroutley** [comment]
Hi everyone! This is my link 
https://tianna338.opened.ca

**Comment by emilylehoux** [comment]
Here is Mine!
https://indigenousstudieseducation.opened.ca/category/edci-338/
